Framework/Geometry/src/Objects/MeshObjectCommon.cpp

```diff
@@ -0,0 +1,286 @@
+#include "MantidGeometry/Objects/MeshObjectCommon.h"
+#include "MantidGeometry/Objects/BoundingBox.h"
```

Comment:
and here

Reply:
I think this should not change as there needs to be exceptions for implementation files of components including their definitions ["A component's .cpp file includes its corresponding .h file as the first substantive line of code"](https://github.com/bloomberg/bde/wiki/physical-code-organization)

John Lakos goes into detail on this. in his "Large Scale C++ Design" book. I'll change it anyway, but maybe this is something for the TSC to discuss.

Reply:
Correct. Sorry for the noise.